web/html/src/components/table/SearchField.tsx

```diff
@@ -1,17 +1,30 @@
 import * as React from "react";
 
+import ReactSelect from "react-select";
```

Comment:
Please prefer `import { Select } from "components/input/Select";` instead, that way we get shared styles, portaling logic etc.

Reply:
hum... this one is for use with `Form` and we don't have one in that page... would that work or should I add a dummy one?

Reply:
Done